diff --git a/node_modules/fantasticon/lib/generators/asset-types/svg.js b/node_modules/fantasticon/lib/generators/asset-types/svg.js
index 8dcc6bc..a16241d 100644
--- a/node_modules/fantasticon/lib/generators/asset-types/svg.js
+++ b/node_modules/fantasticon/lib/generators/asset-types/svg.js
@@ -17,7 +17,11 @@ const generator = {
         for (const { id, absolutePath } of Object.values(assets)) {
             const glyph = (0, fs_1.createReadStream)(absolutePath);
             const unicode = String.fromCharCode(codepoints[id]);
-            glyph.metadata = { name: id, unicode: [unicode] };
+            let ligature = '';
+            for (var i = 0; i < id.length; i++) {
+                ligature += String.fromCharCode(id.charCodeAt(i));
+            }
+            glyph.metadata = { name: ligature, unicode: [unicode, ligature] };
             fontStream.write(glyph);
         }
         fontStream.end();
diff --git a/node_modules/fantasticon/lib/types/runner.d.ts b/node_modules/fantasticon/lib/types/runner.d.ts
index 42e4693..f681280 100644
--- a/node_modules/fantasticon/lib/types/runner.d.ts
+++ b/node_modules/fantasticon/lib/types/runner.d.ts
@@ -26,6 +26,7 @@ export type RunnerOptionalOptions = {
     prefix: string;
     fontsUrl: string;
     getIconId: GetIconIdFn;
+    ligatures: boolean;
 };
 export type RunnerOptionsInput = RunnerMandatoryOptions & Partial<RunnerOptionalOptions>;
 export type RunnerOptions = RunnerMandatoryOptions & Partial<RunnerOptionalOptions>;
